admin/partials/page-single-push.php

```diff
@@ -80,6 +80,15 @@
 					</label>
 				</td>
 			</tr>
+			<tr>
+				<th scope="row"><?php esc_html_e( 'Suppress Video Thumbnail?', 'apple-news' ); ?></th>
+				<td>
+					<label for="apple-news-suppress-video-url">
+						<input id="apple-news-suppress-video-url" name="apple_news_suppress_video_url" type="checkbox" value="1" <?php checked( $post_meta['apple_news_suppress_video_url'][0] ); ?>>
+						<?php esc_html_e( 'Check this to prevent video thumbnails for this article.', 'apple-news' ); ?>
+					</label>
+				</td>
+			</tr>
```

Comment:
Wasn't able to find where `page-single-push` renders on the frontend (somewhere conditionally inside `admin_page()`).

Reply:
This might be legacy - we used to have a way of setting options before push if you went to the article list and clicked Publish, but now Publish just publishes the article directly. At some point we should remove this partial, but not necessary for this PR.